english/grammatical_error_correction.md

```diff
@@ -18,7 +18,7 @@ The shared task setting restricts that systems use only publicly available datas
 
 | Model           | F0.5  |  Paper / Source | Code |
 | ------------- | :-----:| --- | :-----: |
-| T5 ([t5.1.1.xxl](https://github.com/google-research/text-to-text-transfer-transformer/blob/master/released_checkpoints.md)) trained on [cLang-8](https://github.com/google-research-datasets/clang8) (Rothe et al., ACL-IJCNLP 2021) | 68.87 | [A Simple Recipe for Multilingual Grammatical Error Correction](https://arxiv.org/pdf/2106.03830.pdf) | [T5](https://github.com/google-research/text-to-text-transfer-transformer), [cLang-8](https://github.com/google-research-datasets/clang8) |
```

Comment:
The reason is because that current score is not comparable, as discussed here https://github.com/google-research-datasets/clang8/issues/3#issuecomment-991151706
@ekQ Hi Eric, would you mind sharing the T5-xxl's score on the CoNLL-2014 official annotation set?

Reply:
I will do the score moving on a different PR instead, and ask the author of that paper (who is also the one adding the score to this readme file) to review the new PR.

Reply:
Great, thanks!